lib/db/mysql/schema.sql

```diff
@@ -41,7 +41,7 @@ CREATE TABLE IF NOT EXISTS tokens (
   FOREIGN KEY (clientId) REFERENCES clients(id) ON DELETE CASCADE,
   userId BINARY(16) NOT NULL,
   INDEX tokens_user_id(userId),
-  email VARCHAR(256) NOT NULL,
+  email VARCHAR(256),
```

Comment:
IIUC, this puts `schema.sql` out of sync with the migration patches, which restored the `NOT NULL` constrain on this column.